tests/adapter/mongo/test_variant_evaluation_terms.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from scout.adapter.mongo.variant_evaluation_term import EvaluationTerm
+from scout.server.extensions import store
+
+
+def test_evaluation_term_no_key():
+    """Test creating an evaluation term with no key: should raise ValueError"""
+    term_dict = dict(
+        term_label="label",
+        term_description="description",
+        term_category="dismissal_term",
+        term_tracks=["rare"],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_no_label():
+    """Test creating an evaluation term with no label: should raise ValueError"""
+    term_dict = dict(
+        term_key=1,
+        term_description="description",
+        term_category="dismissal_term",
+        term_tracks=["rare"],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_no_description():
+    """Test creating an evaluation term with no description: should raise ValueError"""
+    term_dict = dict(
+        term_key=1,
+        term_label="label",
+        term_category="dismissal_term",
+        term_tracks=["rare"],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_wrong_category():
+    """Test creating an evaluation term with wrong category: should raise ValueError"""
+    term_dict = dict(
+        term_key=1,
+        term_label="label",
+        term_description="description",
+        term_category="FOO",
+        term_tracks=["rare"],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_no_tracks():
+    """Test creating an evaluation term with no tracks: should raise ValueError"""
+    term_dict = dict(
+        term_key=1,
+        term_label="label",
+        term_description="description",
+        term_category="dismissal_term",
+        term_tracks=[],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_wrong_track():
+    """Test creating an evaluation term with non-standard tracks: should raise ValueError"""
+    term_dict = dict(
+        term_key=1,
+        term_label="label",
+        term_description="description",
+        term_category="dismissal_term",
+        term_tracks=["BAR"],
+    )
+    with pytest.raises(ValueError):
+        EvaluationTerm(term_dict)
+
+
+def test_evaluation_term_existing_key(adapter):
```

Comment:
If possible consider adding tests for trying to access non-existent evaluation term

Reply:
Good suggestion! And we should also taking care of flashing a message whenever a variant was previously tagged with a key that is then removed from the database. I think in this situation the app might crash at the moment, I'll check

Reply:
Fixed, I've added tests in tests/scout/server/blueprints/variants/test_variants_controllers